packages/design-system/src/components/Table/Table.test.tsx

```diff
@@ -43,34 +43,10 @@ describe('Table', function () {
     expect(screen.getByRole('table')).toHaveClass('ds-c-table--striped');
   });
 
-  it('accepts custom id', () => {
```

Comment:
Custom `id` seems to be used to tie the caption to the scrollableNotice, but since that was removed, the `id` served no purpose